src/layout/comp/navbar.module.scss

```diff
@@ -57,3 +57,16 @@ progress.progress {
 progress.show {
   display: block;
 }
+
+button.tryTidbCloudBtn {
+  background-color: #3d3fea;
+  color: #fff;
+  border-color: transparent;
+
+  &:hover,
+  &:active,
+  &:focus {
+    color: #fff;
+    border-color: transparent;
```

Comment:
Seems these two attributes doesn't change the button style when hover event triggered.

Reply:
Just for override global hover style here.